code/game/objects/items/devices/uplinks.dm

```diff
@@ -193,6 +193,7 @@ var/list/world_uplinks = list()
 /obj/item/device/radio/uplink/New()
 	hidden_uplink = new(src)
 	icon_state = "radio"
+	hidden_uplink.uses = 30

```

Comment:
Don't do this here, I told you not to do it, and it won't even work. Do it in nuclear.dm post_equip(), chance tc to 30
https://github.com/tgstation/-tg-station/blob/master/code/game/gamemodes/nuclear/nuclear.dm#L313 this line